projects/Mallard/android/app/build.gradle

```diff
@@ -102,10 +102,10 @@ android {
     }
 
     defaultConfig {
-        applicationId "com.mallard"
+        applicationId "com.guardian.editions"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 1
+        versionCode 2
```

Comment:
We have to bump the [`versionCode`](https://developer.android.com/studio/publish/versioning#appversioning) manually at the moment, otherwise the deploy to the Play Store will fail. We should consider how to do this automatically in future.

Reply:
this might help us but it's a separate PR https://github.com/Jems22/fastlane-plugin-increment_version_code